src/Data/Nat/Properties.agda

```diff
@@ -1883,13 +1888,25 @@ m≤∣m-n∣+n m n = subst (m ≤_) (+-comm n _) (m≤n+∣m-n∣ m n)
 ⌊n/2⌋<n zero    = z<s
 ⌊n/2⌋<n (suc n) = s<s (s≤s (⌊n/2⌋≤n n))
 
+a≡⌊a+a/2⌋ : ∀ {a} → a ≡ ⌊ (a + a) /2⌋
```

Comment:
Could we make the argument to these two proofs explicit rather than implicit? I know they can be inferred most of the time, but it will make proofs using them easier to read.

Reply:
Done!